third_party/libpg_query/include/parser/kwlist.hpp

```diff
@@ -326,6 +326,7 @@ PG_KEYWORD("read", READ_P, UNRESERVED_KEYWORD)
 PG_KEYWORD("real", REAL, COL_NAME_KEYWORD)
 PG_KEYWORD("reassign", REASSIGN, UNRESERVED_KEYWORD)
 PG_KEYWORD("recheck", RECHECK, UNRESERVED_KEYWORD)
+PG_KEYWORD("recurring", RECURRING, RESERVED_KEYWORD)
```

Comment:
It seems we're now using a new syntax (`RECURRING tbl`) instead of using the `recurring_[tbl]` in the PR description. I'm not specifically opposed to this - but this is adding a new reserved keyword, which is a breaking change if people use the word `recurring` as a column identifier, type name or function name. 

Given that CTEs cannot be scoped in a schema, could I suggest the syntax `recurring.cte` instead? That way `RECURRING` does not need to be a reserved keyword.